Makefile

```diff
@@ -73,6 +73,11 @@ ifeq ($(UNAME_O),Cygwin)
 	LIBS += -liconv
 	EXT = .exe
 endif
+ifeq ($(UNAME_O),Msys)
+	CFLAGS += -I/mingw64/include -I/mingw64/lib/libzip/include
+	LIBS += -liconv -llibzip -L/mingw64/lib -lzip -lz -L/mingw64/lib -L/mingw64/bin
```

Comment:
/mingw64/bin has was a try for libzip.dll present there. 

Will change all that stuff and try again. Since msys packages has CI is easy to see if it works (not only on my laptop)